04_itn-eval-metrics_gsheetInfo.Rmd

```diff
@@ -261,16 +265,20 @@ nrow(unique_emails)
 ### Get career stage of workshop registrants
 ```{r}
 career_stage_counts <- readr::read_tsv(file.path("data", "career_stage_counts.tsv"))
+
 career_stage_counts_subset <- career_stage_counts[c(1:5), c(1:11)]
-df_to_plot <- data.frame(Stage = colnames(career_stage_counts_subset[2:10]), count = unlist(career_stage_counts_subset[5, c(2:10)]))
-df_to_plot <- data.frame(Stage = colnames(career_stage_counts_subset[2:10]), count = unlist(career_stage_counts_subset[5, c(2:10)]), "Trainee" = c("yes","yes","no","no","no","yes","no","yes","yes"))
-ggplot(df_to_plot, aes(x=reorder(Stage, -count), y=count, fill=Trainee)) +
+
+career_stage_counts_final <- data.frame(Stage = colnames(career_stage_counts_subset[2:10]), count = unlist(career_stage_counts_subset[5, c(2:10)]), "Trainee" = c("yes","yes","no","no","no","yes","no","yes","yes"))
```

Comment:
I have the same comment about this data frame as I did here https://github.com/FredHutch/itn-dashboard/pull/7#discussion_r1499218080

I'm a tad worried this code isn't super reproducible or robust to changes in the dataset. 

However, because this repo will eventually be deprecated, I wouldn't worry about changing this here. But I would like us to make this part a little more robust on the itn-dashboard repo.